crates/rpc/rpc/src/eth/api/optimism.rs

```diff
@@ -28,3 +30,16 @@ impl OptimismTxMeta {
         Self { l1_block_info, l1_fee, l1_data_gas }
     }
 }
+
+/// Optimism Sequencer Client
+#[derive(Debug, Default, Clone)]
+pub struct SequencerClient {
```

Comment:
this should be moved to the optimism.rs file

Reply:
Do you mean to this file? https://github.com/paradigmxyz/reth/blob/4559717096da734398e97c56599e7e027b61a775/crates/primitives/src/transaction/optimism.rs#L4